package.json

```diff
@@ -23,11 +32,13 @@
     "eslint-plugin-react": "^7.32.2",
     "eslint-plugin-react-hooks": "^4.6.0",
     "fs-extra": "^10.1.0",
+    "lodash-es": "^4.17.21",
     "prompts": "^2.4.2",
     "ts-node": "^10.9.1",
     "typescript": "^4.9.5"
   },
   "scripts": {
-    "newHook": "ts-node scripts/newHook.ts"
+    "newHook": "ts-node scripts/newHook.ts",
+    "lint": "eslint ."
   }
```

Comment:
It will broken the script, fix it.  You can try to add a hook like useCookie for test.

Reply:
Sure